Xamarin.PropertyEditing.Mac/Controls/BasePathEditorControl.cs

```diff
@@ -92,7 +86,13 @@ private void CurrentTextField_Changed (object sender, EventArgs e)
 		private void BrowsePathButton_Activated (object sender, EventArgs e)
 		{
 			Window.MakeFirstResponder (this.currentTextField);
-			this.panel.BeginSheet (this.Window, HandleAction);
+
+			this.panel.AllowsMultipleSelection = false;
```

Comment:
This was the properties required for Browse to work will always be set correctly, before we open the panel.